packages/api/bin/serveUtils.js

```diff
@@ -57,13 +57,11 @@ async function addCollections(collections) {
     },
   });
 
-  const collectionModel = new models.Collection();
   const es = await getESClientAndIndex();
   const collectionPgModel = new CollectionPgModel();
   return await Promise.all(
     collections.map(async (c) => {
-      const dynamoRecord = await collectionModel.create(c);
-      await indexer.indexCollection(es.client, dynamoRecord, es.index);
+      await indexer.indexCollection(es.client, c, es.index);
```

Comment:
Nit, and not anything you did: can we make `c` `collection`?     One letter vars most of the time, even in .map are squicky.